packages/graphql-codegen-cli/src/config.ts

```diff
@@ -445,3 +445,19 @@ function addHashToDocumentFiles(documentFilesPromise: Promise<Types.DocumentFile
     })
   );
 }
+
+export function shouldEmitLegacyCommonJSImports(config: Types.Config, outputPath: string): boolean {
+  const globalValue = config.emitLegacyCommonJSImports === undefined ? true : !!config.emitLegacyCommonJSImports;
+  // const outputConfig = config.generates[outputPath];
+
+  // if (!outputConfig) {
```

Comment:
@saihaj we need to discuss whenever we want `emitLegacyCommonJSImports` to be also set at the GenerateConfig level 👀

Reply:
Discussed together via Slack, let's keep it simple for now.